.github/workflows/analyze-and-build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         with: {ruby-version: ^2.6}
       - uses: actions/checkout@master
       - run: yarn install --frozen-lockfile
-      - run: gem install bundler && bundle check || bundle install --frozen --path vendor/bundle
+      - run: gem install bundler:1.17.2 && bundle check || bundle install --frozen --path vendor/bundle
```

Comment:
This certainly makes sense to me. Until Fastlane supports v2 or whatever the problem is preventing us from upgrading, we should pin.